### ISSUE TITLE
Analyzer not taking into account wildcard / analyze_by_access_level

### ISSUE THREAD

**Issue by 5vmmvm** [salesforce/policy_sentry#162]
Hi team,

I was testing the last version of the analyzer and it seems the wildcard is not taken into account to be analyzed for an action (it was in the past). I am attaching a simple code to show that it's not recognizing the potential permissions s3:GetObjects*, s3:PutObjects*.

Thanks a lot for your work.

```
#!/usr/bin/env python
from policy_sentry.analysis.analyze import analyze_by_access_level
import json

if __name__ == '__main__':
    #mha_pivot template
    permissions_management_policy = {
        "Statement": [
            {
            "Action": [
                "s3:GetObject*",
                "s3:PutObject*"
            ],
            "Effect": "Allow",
            "Resource": [
                "*"
            ]
            }
        ],
        "Version": "2012-10-17"
        }

    print('********* READ ***********')
    permissions_management_actions = analyze_by_access_level(permissions_management_policy, "Read")
    print(json.dumps(permissions_management_actions, indent=4))
    print('********* LIST ***********')
    permissions_management_actions = analyze_by_access_level(permissions_management_policy, "List")
    print(json.dumps(permissions_management_actions, indent=4))
    print('********* WRITE ***********')
    permissions_management_actions = analyze_by_access_level(permissions_management_policy, "Write")
    print(json.dumps(permissions_management_actions, indent=4))
    print('********* TAGGING ***********')
    permissions_management_actions = analyze_by_access_level(permissions_management_policy, "Tagging")
    print(json.dumps(permissions_management_actions, indent=4))
    print('********* PERMISSIONS-MANAGEMENT ***********')
    permissions_management_actions = analyze_by_access_level(permissions_management_policy, "Permissions management")
    print(json.dumps(permissions_management_actions, indent=4))
```

**Comment by kmcquade** [comment]
@5vmmvm  - thanks for submitting this. I fixed this #165 

Glad you are still making use of those analysis functions. Note that in this PR, I made a few changes that *might* affect you. The changes are noted in this issue: #164 

You'll probably enjoy the new `get_expanded_policy`, which accepts a polciy as a dictionary, and expands the * actions in a single line. It's easier than what was previously in place.

I added your example as a unit test and verified that it passes. As an FYI, the output is shown below, which shows that the issue is fixed by that PR

```
Launching unittests with arguments python -m unittest test_analyze.AnalysisExpandWildcardActionsTestCase.test_gh_162 in [redacted]/policy_sentry/test/analysis


Process finished with exit code 0
********* READ ***********
[
    "s3:GetObject",
    "s3:GetObjectAcl",
    "s3:GetObjectLegalHold",
    "s3:GetObjectRetention",
    "s3:GetObjectTagging",
    "s3:GetObjectTorrent",
    "s3:GetObjectVersion",
    "s3:GetObjectVersionAcl",
    "s3:GetObjectVersionForReplication",
    "s3:GetObjectVersionTagging",
    "s3:GetObjectVersionTorrent"
]
********* LIST ***********
[]
********* WRITE ***********
[
    "s3:PutObject",
    "s3:PutObjectLegalHold",
    "s3:PutObjectRetention"
]
********* TAGGING ***********
[
    "s3:PutObjectTagging",
    "s3:PutObjectVersionTagging"
]
********* PERMISSIONS-MANAGEMENT ***********


Ran 1 test in 0.143s

OK
[
    "s3:PutObjectAcl",
    "s3:PutObjectVersionAcl"
]
```

**Comment by 5vmmvm** [comment]
Awesome! thanks
